charactersheet/charactersheet/utilities/fixtures.js

```diff
@@ -142,20 +142,50 @@ var Fixtures = {
         spellcastingAbilityOptions: [
             'INT', 'WIS', 'CHA']
     },
-    weapon : {
-        weaponProficiencyOptions : [
-            'Simple', 'Martial', 'Improvised', 'Nonlethal', 'Exotic'],
-        weaponHandednessOptions : [
-            'Light', 'One-Handed', 'Two-Handed'],
-        weaponTypeOptions : [
-            'Melee', 'Ranged'],
-        weaponSizeOptions : [
-            'Small', 'Medium', 'Large'],
-        weaponPropertyOptions : [
-            'Ammunition', 'Finesse', 'Heavy', 'Light', 'Loading',
-            'Range', 'Reach', 'Special', 'Thrown', 'Versatile'],
-        weaponDamageTypeOptions : [
-            'Bludgeoning', 'Piercing', 'Slashing']
+    weapon: {
+        weaponDamageTypeOptions: [
```

Comment:
These properties don't match with the SRD data, which causes the field to not populate.